hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/dt-1.11.5/css/jquery.dataTables.css

```diff
@@ -1,21 +1,3 @@
-/**
```

Comment:
Do we need this? I mean is it OK to add "The ASF licenses this file to you under the Apache License, Version 2.0 (the "License")" if the JS pair of this file this
`opyright 2008-2021 SpryMedia Ltd.

 This source file is free software, available under the following license:
   MIT license - http://datatables.net/license
`

Reply:
We shouldn't remove it, however I'm not sure about the second part of the sentence, what do you mean by "if the JS pair of this file this"?

Reply:
OK, i will revert this delete. The [jquery.dataTables.min.js](https://github.com/apache/hadoop/blob/537ca4d321b6cc8c69326af7e1c7dc75177f2fd5/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/webapps/static/dt-1.11.5/js/jquery.dataTables.min.js) file starts with an MIT licence.